web_programming/reddit_crawler_hot_topics.py

```diff
@@ -0,0 +1,33 @@
+import praw
+
+# creating an instance for praw reddit
+# get all the user cridentials form the reddit page: https://www.reddit.com/prefs/apps
+reddit = praw.Reddit(
+    # Client ID
+    client_id="Enter client id here",
+    # Client Secret Key

```

Comment:
The code already says this so the comments can be removed.

Reply:
ok